pkg/network/multus/nad.go

```diff
@@ -51,19 +51,27 @@ func NetAttachDefNamespacedName(namespace, fullNetworkName string) types.Namespa
 	}
 }
 
-func GetNetworkToResourceMap(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachineInstance) (networkToResourceMap map[string]string, err error) {
-	networkToResourceMap = make(map[string]string)
+func GetNetworkToResourceMap(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachineInstance) (map[string]string, error) {
+	networkToResourceMap := map[string]string{}
+
 	for _, network := range vmi.Spec.Networks {
-		if network.Multus != nil {
-			nadNamespacedName := NetAttachDefNamespacedName(vmi.Namespace, network.Multus.NetworkName)
-			crd, err := virtClient.NetworkClient().K8sCniCncfIoV1().NetworkAttachmentDefinitions(nadNamespacedName.Namespace).Get(context.Background(), nadNamespacedName.Name, metav1.GetOptions{})
-			if err != nil {
-				return map[string]string{}, fmt.Errorf("failed to locate network attachment definition %s", nadNamespacedName.String())
-			}
-			networkToResourceMap[network.Name] = GetResourceNameForNetwork(crd)
+		if network.Multus == nil {
+			continue
+		}
+
+		nadNamespacedName := NetAttachDefNamespacedName(vmi.Namespace, network.Multus.NetworkName)
+		netAttachDef, err := virtClient.NetworkClient().
+			K8sCniCncfIoV1().
+			NetworkAttachmentDefinitions(nadNamespacedName.Namespace).
+			Get(context.Background(), nadNamespacedName.Name, metav1.GetOptions{})
+		if err != nil {
+			return map[string]string{}, fmt.Errorf("failed to locate network attachment definition %s", nadNamespacedName.String())
```

Comment:
nit: Since an error has occurred, wouldn't it be more idiomatic to return nil instead of an empty map?

Reply:
Done.